### PR TITLE
fix bug for multiple-output not correct in Random Forest

### PR DESCRIPTION
**Pull request by xiaowuhu**
Signed-off-by: xiaowuhu <xiaowuhu@microsoft.com>

https://github.com/onnx/sklearn-onnx/issues/889

will add more test cases later

### CODE REVIEW

File: .azure-pipelines/win32-conda-CI.yml [hunk 1]
```diff
@@ -112,16 +112,6 @@ jobs:
         onnxrt.version: 'onnxruntime==1.7.0'  # -i https://test.pypi.org/simple/ ort-nightly'
         onnxcc.version: 'onnxconverter-common==1.8.1'  # git+https://github.com/microsoft/onnxconverter-common.git
         sklearn.version: '==0.24.2'
-      Py38-Onnx181-Rt170-Skl0241:
-        python.version: '3.8'
```
Comment (xadupre) [inline .azure-pipelines/win32-conda-CI.yml:116]: Python 3.8 breaks?

Comment (xiaowuhu) [reply .azure-pipelines/win32-conda-CI.yml:116]: yes. the log is [here](https://dev.azure.com/onnxmltools/bafa8fda-79b9-45f5-8ee6-97f3d532fc46/_apis/build/builds/22385/logs/263).
it says cannot import apply_reducesum. I don't know how to fix it.

Comment (xadupre) [reply .azure-pipelines/win32-conda-CI.yml:116]: You should update onnxconverter-common==1.7.0 to onnxconverter-common==1.8.1, that should fix it.

Comment (xiaowuhu) [reply .azure-pipelines/win32-conda-CI.yml]: thanks @xadupre for the hint. it works.

**Review by xadupre — APPROVED** [review]

**Pull request merged**